build/build.rs

```diff
@@ -40,3 +57,55 @@ fn main() {
         println!("cargo:rustc-link-lib=c++_shared");
     }
 }
+
+fn build_windows_i686() {
+    // Prepare directories
+    let cargo_dir = Path::new(env!("CARGO_MANIFEST_DIR"));
+    let source_dir = cargo_dir.join("glslang");
+    let install_dir = source_dir.join("install");
+    let library_source = install_dir.join("lib");
+    let library_destination = cargo_dir.join("build").join("windows-i686");
+
+    // Re-use libraries if they exist
+    if library_destination.exists() {
+        return;
+    }
+
+    // Initialize submodules
+    Command::new("git")
+        .args(&["submodule", "update", "--init"])
```

Comment:
The published crate won't contain a `.git` directory, so this will fail. It shouldn't be needed in that case anyway for as long as this is run before publishing.

Reply:
Does this mean the submodule is included in the crate, or do you mean the compiled libraries?

Reply:
If `git submodule update --init` was run prior to publishing, it the submodule source should be included in the crates.io package as is without being a submodule.